packages/react/src/components/Table/TableRow.tsx

```diff
@@ -75,7 +77,25 @@ class TableRow extends UIComponent<WithAsProp<TableRowProps>, any> {
     accessibility: tableRowBehavior as Accessibility,
   }
 
-  renderCells(variables: ComponentVariablesObject) {
+  rowRef = React.createRef<HTMLElement>()
+
+  actionHandlers = {
+    unsetRowTabbable: e => {
+      this.rowRef.current.setAttribute('tabindex', '-1')
```

Comment:
we should find a better way to do this, let's talk

Reply:
bug reported please see comment above